tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -852,7 +859,40 @@ const reducer = createReducer(
         }
       }
 
-      const nextState = {
+      if (response.runToSeries) {
+        for (let runId in response.runToSeries) {
+          const cardMetadata: CardMetadata = {
+            plugin,
+            tag,
+            runId,
```

Comment:
Scalar cards should have runId null since they are "multi-run" cards. If you look at the state constructed for a scalar card with 4 runs, for example, you see that it gets 4 different entries in state.cardToTimeSelection. It should have a single entry.

See: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/store/metrics_reducers.ts;l=120;drc=99e0e2788ee9f0bb95b3e08e5c018e394e3a18c8

If you need to construct individual cardMetadata objects, maybe its worth refactoring the logic out of buildCardMetadataList rather than trying to replicate it here.